ocfweb/about/templates/about/staff.html

```diff
@@ -50,14 +50,14 @@
     <div class="ocf-services">
         <div class="container">
             <div class="row">
-                <h2>OCF staff help to provide cool services to the Berkeley community.</h2>
+                <h2>OCF staff help provide cool services to the Berkeley community.</h2>
             </div>
             <div class="row">
                 <div>
                     <div class="col-sm-4">
                         <p><img src="{% static 'img/about/web.png' %}" alt="web hosting" /></p>
                         <h3>Web &amp; Email Hosting</h3>
-                        <p>We provide free web and email hosting to our more than 40,000 members, both students and student groups.</p>
+                        <p>We provide free web and email hosting to more than 40,000 of our members, both students and student groups.</p>
```

Comment:
This changes the meaning. The fact is we _have_ more than 40 000 members, not that we provide web/email hosting to all of them.